hcn/hcnendpoint_test.go

```diff
@@ -309,3 +309,57 @@ func TestModifyEndpointSettings(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestApplyTierAclPolicyOnEndpoint(t *testing.T) {
+	network, err := HcnCreateTestL2BridgeNetwork()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
+		err = network.Delete()
+		if err != nil {
+			fmt.Printf("Failed deleting from defer routine network: %s-%s \n", network.Id, network.Name)
+			t.Fatal(err)
```

Comment:
I'd prefer if the error just had all of the information instead of writing to stdout right before. Same for line 333-334

```go
t.Fatalf("failed to cleanup l2bridge network %s-%s: %v", network.Id, network.Name, err)
```